waltid-services/waltid-verifier-api/src/main/kotlin/id/walt/verifier/oidc/OIDCVerifierService.kt

```diff
@@ -149,7 +150,7 @@ object OIDCVerifierService : OpenIDCredentialVerifier(
     // Simple cryptographic operations interface implementation
     override fun doVerify(tokenResponse: TokenResponse, session: PresentationSession): Boolean {
         val policies = sessionVerificationInfos[session.id]
-            ?: throw IllegalArgumentException("Could not find policy listing for session: ${session.id}")
+            ?: throw NotFoundException("Could not find policy listing for session: ${session.id}")
```

Comment:
I was wondering. Is there anything the user can do if the policy listing could not be found? If yes, ok. But, if not, why should we tell him this?

Reply:
I think if im not mistaken those policies are pre-configured in the verifier-api .
If for some reason a user want to use a non existing policy it will throw this error. In this case the user will have to implement this new policy or use one of the existing once ex : `expired": "Verifies that the credentials expiration date (`exp` for JWTs) has not been exceeded."`